test/helpers/overlays.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2021 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { OverlayEdgePosition, OverlayShapePosition } from '../../src/component/registry';
+
+export const overlayEdgePositionValues = <Array<OverlayEdgePosition>>['start', 'middle', 'end'];
+
+export const overlayShapePositionValues = <Array<OverlayShapePosition>>[
```

Comment:
I think to avoid to have this method, and to find all the value manually for the HTML page, it's better to have an enum like FitType.

- With enum: https://github.com/process-analytics/bpmn-visualization-js/blob/master/src/static/js/rendering-diagram.js#L155-L157
- Without enum: https://github.com/process-analytics/bpmn-visualization-js/blob/master/src/static/js/overlays.js#L135-L145

Reply:
This function is only used to avoid duplication in tests. Integration won't probably need to put an overlay on all positions for real use cases. So to me this is not an issue.
We already discussed about the drawbacks of using enums. I don't remember exactly why we decided to put a string type union here. For pros/cons, we can check https://2ality.com/2020/02/enum-alternatives-typescript.html#type-unions-vs.-enums
Anyway, if you think this change is valuable, we can think about it but not in this PR IMHO: it is already quite big and this change would break the API as well.